CondFormats/EcalObjects/src/classes.h

```diff
@@ -222,5 +223,14 @@ namespace CondFormats_EcalObjects {
     EcalContainer<EBDetId,EcalPulseSymmCovariance> ec_ebDetId_ecalPulseSymmCovariance;
     EcalCondObjectContainer<EcalPulseSymmCovariance> ecalSPCmap; //typedef EcalPulseSymmCovariance
 
+    EcalSimPulseShape ecal_sim_pulse_shapes;
+    std::vector<double> v_barrel_shape;
+    std::vector<double> v_endcap_shape;
+    std::vector<double> v_apd_shape; 
+    double barrel_thresh;
+    double endcap_thresh;
+    double apd_thresh;
+    float time_interval; 
```

Comment:
these basic types  don't seem to belong in this file.
I'd expect only EcalSimPulseShape with whatever containers using/needing it.

Reply:
thanks for the catch. this is now cleaned up. I will update the PR soon.